pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/AvslagForLiteTrygdetidAP.kt

```diff
@@ -0,0 +1,547 @@
+                            English to "This decision was made pursuant to the provisions of §§ 19-4 and 20-2 of the National Insurance Act.",
+                        )
+                    }
+                }
+                showIf(erAp2025) {
+                    paragraph {
+                        text(
+                            Bokmal to "Vedtaket er gjort etter folketrygdloven § 20-2.",
+                            Nynorsk to "Vedtaket er gjort etter folketrygdlova § 20-2.",
+                            English to "This decision was made pursuant to the provisions of § 20-2 of the National Insurance Act.",
+                        )
+                    }
+                }
+                title1 {
+                    text(
+                        Bokmal to "Du kan selv sjekke når du kan få alderspensjon",
+                        Nynorsk to "Du kan sjølv sjekke når du kan få alderspensjon",
+                        English to "You can find out when you are eligible for retirement pension",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Du kan ta ut alderspensjon før du fyller 67 år hvis du har hatt høy nok pensjonsopptjening. I Din pensjon på $DIN_PENSJON_URL kan du se hva pensjonen din blir, avhengig av når og hvor mye du tar ut. Slik kan du sjekke når du tidligst kan ta ut alderspensjon.",
+                        Nynorsk to "Du kan ta ut alderspensjon før du fyller 67 år dersom du har hatt høg nok pensjonsopptening. I Din pensjon på $DIN_PENSJON_URL kan du sjå kva pensjonen din blir, avhengig av når og kor mykje du tek ut. Slik kan du sjekke når du tidlegast kan ta ut alderspensjon.",
+                        English to "You may be eligible for retirement pension before the age of 67, provided your accumulated pension capital is sufficiently high. Log on to Din pensjon at $DIN_PENSJON_URL to find out more about your pension payments. You can also see how your payments change depending on when you start drawing a retirement pension and what percentage of retirement pension you choose."
+                    )
+                }
+
+                paragraph {
+                    text(
+                        Bokmal to "Du må sende oss en ny søknad når du ønsker å ta ut alderspensjon. En eventuell endring kan tidligst skje måneden etter at vi har mottatt søknaden.",
+                        Nynorsk to "Du må sende oss ein ny søknad når du ønskjer å ta ut alderspensjonen. Ei eventuell endring kan tidlegast skje månaden etter at vi har mottatt søknaden.",
+                        English to "You have to submit an application when you want to start drawing your retirement pension. Any change will be implemented at the earliest the month after we have received the application."
+                    )
+                }
+            }
+            title1 {
```

Comment:
I doksys er include condition for SU "if(	(faktiskBostedsland_ == ""
|| faktiskBostedsland_ == "NORGE"
)
&&	(
begrunnelse == "UNDER_1_AR_TT"
||	begrunnelse == "UNDER_3_AR_TT"
||	begrunnelse == "UNDER_5_AR_TT"
)	)
ReturnValue("3");"